### ISSUE TITLE
How to plot my overlap

### ISSUE THREAD

**Issue by Oliveira-D-R** [danlwarren/ENMTools#230]
Hello to all ENMtools users!

I have a question about plotting the overlap of two species. I'm not doing a species distribution model, so I'm working with enmtools.ecospat.bg and enmtools.ecospat.id. However, I don't know how I can plot an overlay graph similar to the one in the image below. Is there a code for this in ENMtools?

![image](https://user-images.githubusercontent.com/82961766/221376828-2a8bf581-56a0-4cf3-b1b7-8434fe323758.png)

**Comment by danlwarren** [comment]
I don't think we really have anything like that at the moment.  We might be able to add something in the future, but at the moment we're just frantically trying to update the package to work with terra so we can get it back on CRAN.
